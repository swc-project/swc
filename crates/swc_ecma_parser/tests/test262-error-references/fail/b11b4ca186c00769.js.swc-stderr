
  x Expected unicode escape
   ,-[$DIR/tests/test262-parser/fail/b11b4ca186c00769.js:1:1]
 1 | \\
   : ^
   `----
