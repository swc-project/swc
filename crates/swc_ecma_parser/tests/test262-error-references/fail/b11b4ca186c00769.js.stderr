
  x Expected unicode escape
   ,-[$DIR/tests/test262-parser/fail/b11b4ca186c00769.js:2:1]
 2 | \\
   : ^
   `----
